feature-libs/organization/administration/core/services/org-unit.service.ts

```diff
@@ -497,4 +497,8 @@ export class OrgUnitService {
   ): Observable<StateUtils.LoaderState<B2BUnit>> {
     return this.store.select(getOrgUnitState(orgUnitId));
   }
+
+  isUpdatingUnitAllowed(): boolean {
```

Comment:
Recommended to make it `protected`

Reply:
if we make 'isUpdatingUnitAllowed' protected , it will be accessible only within class 'OrgUnitService' and its subclasses. We wont be able to call it from unit-details.component.ts. Hence not making the change.

Reply:
noted. 
This would allow other components to change the behaviour and thus update the units, even if we disable it.